src/components/header.js

```diff
@@ -6,6 +6,7 @@ const Header = ({children, style}) => {
 		<AppBar 
 			style={style}
 			title={children}
+			titleStyle={ { position: "absolute", right: "10px" } }
```

Comment:
The logo?
![image](https://cloud.githubusercontent.com/assets/357481/13617344/4759d994-e54c-11e5-9845-14bbe87f14aa.png)

Sure. Do you like the placement of the logo and the title?